help/actions/_posts/2015-02-09-export.md

```diff
@@ -6,7 +6,51 @@ order: 3
 
 ### Export Formats
 
-Use this button to export your search results in different formats.  Currently available options are ADS Classic, BibTeX, AASTeX and Endnote.
+Use this button to export your search results in different formats.  Currently available options are:
+###### 1. Tagged Formats:
+* BibTeX
+* BibTeX with abstracts
+* Generic tagged abstracts
+* EndNote format
+* ProCite format
+* Refman format
+* RefWorks format
+* MEDLARS format
+
+###### 2. Latex Formats:
```

Comment:
Use the "official" capitalization for this: LaTeX

Reply:
done.